### PR TITLE
Add javascript for mobile version hamburger menu

### PR DESCRIPTION
**Pull request by batoolfatima2135**
In this branch, I have :

1. Added javascript linter configurations
2. Added javascript for the hamburger menu.
3. Design mobile menu as given in the figma template.

### CODE REVIEW

**Review by joel-kalema — COMMENTED** [review]
Hi Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request  🛑 🛑 

- [ ] You have submitted a project that your partner did not contribute 😭 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by joel-kalema — COMMENTED** [review]
Hi Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request  🛑 🛑 

- [ ] You have submitted a project that your partner did not contribute to. This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#work-and-submission-mode).  😭 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by LionRouge1 — COMMENTED** [review]
Hi @batoolfatima2135, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is supposed to be a pair-programming activity, but you're the only one who contributed to the project. Kindly make sure you work with your partner to submit this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by batoolfatima2135** [comment]
Hey .. this is my solo activity, not pair programming. I don't have partner
for this week

**Comment by batoolfatima2135** [comment]
I don't have any programming partners! I am doing solo activities this week.
attaching the screenshot of my mail from microverse. Microverse has allowed me solo activities this week.
solo

File: mobile-nav.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  x.style.display = 'none';
+});
+
+hamburger.addEventListener('click', () => {
+  const x = document.getElementById('nav');
+  if (x.style.display === 'none') {
+    x.style.display = 'flex';
+  } else {
+    x.style.display = 'none';
+  }
```
Comment (Zuhaib042) [inline mobile-nav.js]: - [ ] There are some issues in your design. kindly consider fixing them as per requirements.
   - When the user clicks on hamburger-menu, then the hamburger should disappear and the cross should appear in its place. but in your design user can see and access both which is wrong so consider fixing it.
 your design
re


figma design
qw

File: index.html [hunk 2]
```diff
@@ -15,14 +16,28 @@
...
                 <nav class="normal_button">
-                    <img class="menu-icon" src="images/Icon - Menu.png" width="24px" height="24px" alt="Menu-Icon">
+
                     <ul class="desktop-menu">
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="#">Contact</a></li>
+                        <li><a href="#Portfolio">Portfolio</a></li>
+                        <li><a href="#About">About</a></li>
+                        <li><a href="#Contact">Contact</a></li>
```
Comment (Zuhaib042) [inline index.html:23]: - [ ] Another issue in your mobile-menu is that when user click on hamburger button, the rest of the page is still open to the user and he can scroll as well, this is something against the design as we can see in the figma template. you can achieve that by applying extra CSS given below by me and also you need to add `overflow-y: hidden;` to `body` when user clicks on hamburger, with javascript and you would revert that styling when user clicks on cross or any other link again.
css given by me
a

figma design
qw

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @batoolfatima2135 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Professional Readme
✅ Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @batoolfatima2135, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**